### PR TITLE
Update macros.md

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: docs/docs/reference/metaprogramming/macros.md [hunk 1]
```diff
@@ -62,14 +61,12 @@ Quotes and splices can also be applied directly to identifiers. An identifier
...
 is treated as a quote `'{x}`. See the Syntax section below for details.
 
-Quotes and splices are duals of each other. For arbitrary
-expressions `e` and types `T` we have:
+Quotes and splices are duals of each other.
+For arbitrary expressions `e` we have:
 
 ```scala
 ${'{e}} = e
 '{${e}} = e
```
Comment (b-studios) [inline docs/docs/reference/metaprogramming/macros.md:69]: Is this now actually true? At some point, some normalization was observable .

File: docs/docs/reference/metaprogramming/macros.md [hunk 2]
```diff
@@ -78,7 +75,7 @@ The type signatures of quotes and splices can be described using
 two fundamental types:
 
 - `Expr[T]`: abstract syntax trees representing expressions of type `T`
-- `Type[T]`: type structures representing type `T`.
+- `Type[T]`: non erased representation of type `T`.
```
Comment (b-studios) [inline docs/docs/reference/metaprogramming/macros.md:78]: Above in line 48 you say "type structure"

**Review by b-studios — APPROVED** [review]

File: docs/docs/reference/metaprogramming/macros.md (hunk 1 above)
Comment (nicolasstucki) [reply docs/docs/reference/metaprogramming/macros.md:69]: That was in the patterns, not here.

File: docs/docs/reference/metaprogramming/macros.md (hunk 2 above)
Comment (nicolasstucki) [reply docs/docs/reference/metaprogramming/macros.md:78]: Yes, there is much more to improve in this document. Might need a full rewrite.

**Pull request merged**